earth_enterprise/docker/README.md

```diff
@@ -0,0 +1,167 @@
+# Docker Image for Running GE Open Source Earth Server and Fusion Server
+
+## Setting up Docker
+
+### Ubuntu
+
+Install Docker as root:
+
+```BASH
+# apt-get install docker.io
+```
+
+Add users that you want to have access to Docker to the `docker` group:
+
+```BASH
+# adduser muser docker
+```
+
+
+## Pulling an Image from Docker Cloud
+
+You can get a pre-built image from a Docker registry.
+
+```BASH
+docker pull thermopylae/opengee-experimental
+```
+
+
+## Running the Container
+
+### Running the Open GEE Servers
+
+#### Running Open GEE Server with Persistent Globe and Map Databases
+
+In order to be able to build and serve Earth databases that persist between 
+instantiations of the Docker image, you need a persistent assets directory 
+outside of the Docker container.
+
+To run the Docker container using a persistent globe assets directory outside 
+the container, use the following script.  Set the `HOST_GEVOL_PATH`
+environment variable to the path outside the container where you want to store
+your persistent assets.
+
+```BASH
+HOST_GEVOL_PATH=/persistent/gevol/ ./bin/start-docker-image.sh
+```
+
+You can optionally set the `GEE_SERVER_PORT` and even  `IMAGE_TAG` to override
+the default values.
+
+
+##### Initializing a New Location for Persistsing Databases
+
+If you want to create the directory structure needed for storing globe and map
+databases in a new location, you can use the following commands (after
+substituting the desired path), and follow the prompts.
+
+```BASH
+sudo mkdir -p /persistent/gevol/assets
+docker run --cap-add=DAC_READ_SEARCH -v /persistent/gevol:/gevol -ti opengee-experimental /opt/google/bin/geconfigureassetroot --new --assetroot /gevol/assets
+```
+
+
+#### Running Open GEE Server with Temporary Globe and Map Databases
+
+To run the Docker container using temporary assets that disapper when the 
+container is stopped, leave the `HOST_GEVOL_PATH` environment valiable empty.
+This is the default behavior
+
+```BASH
+HOST_GEVOL_PATH='' ./bin/start-docker-image.sh
+```
+
+
+#### Running the Fusion UI in the Docker Container
+
+To start the Fusion UI in the Open GEE Docker container, you need to have
+the GEE servers running in the container, e.g. by running
+`./bin/start-docker-image.sh`, and an X11 server outside the container that
+the Fusion UI can connect to.  If you started the container on a Linux 
+distribution running X11 listening for socket connections to `/tmp/.X11-unix`,
+you should only need to run the following command to bring up the Fusion UI:
+
+```BASH
+./bin/attach-fusion.sh
```

Comment:
I was able to get the start script to work by changing the container id to "thermopylae/opengee-experimental". But, in this script, it doesn't matter what I set container id to, I still get:
`attach-fusion.sh: Couldn't find container for image with tag thermopylae/opengee-experimental.`